library/src/main/java/com/getstream/sdk/chat/adapter/viewholder/attachment/AttachmentViewHolder.kt

```diff
@@ -1,46 +1,38 @@
-package com.getstream.sdk.chat.adapter
+package com.getstream.sdk.chat.adapter.viewholder.attachment
 
 import android.content.Context
 import android.view.View
 import android.view.ViewGroup
 import android.widget.AdapterView.OnItemClickListener
 import android.widget.AdapterView.OnItemLongClickListener
-import android.widget.ListView
-import android.widget.TextView
 import androidx.constraintlayout.widget.ConstraintLayout
 import androidx.core.view.isVisible
 import androidx.core.view.updateLayoutParams
 import com.bumptech.glide.Glide
 import com.getstream.sdk.chat.Chat.Companion.getInstance
 import com.getstream.sdk.chat.R
+import com.getstream.sdk.chat.adapter.AttachmentListAdapter
+import com.getstream.sdk.chat.adapter.AttachmentListItem
 import com.getstream.sdk.chat.adapter.MessageListItem.MessageItem
+import com.getstream.sdk.chat.adapter.inflater
+import com.getstream.sdk.chat.databinding.StreamItemAttachMediaBinding
+import com.getstream.sdk.chat.databinding.StreamItemAttachmentBinding
 import com.getstream.sdk.chat.model.ModelType
 import com.getstream.sdk.chat.utils.LlcMigrationUtils
-import com.getstream.sdk.chat.utils.roundedImageView.PorterShapeImageView
 import com.getstream.sdk.chat.view.MessageListView.AttachmentClickListener
 import com.getstream.sdk.chat.view.MessageListView.BubbleHelper
 import com.getstream.sdk.chat.view.MessageListView.MessageLongClickListener
 import com.getstream.sdk.chat.view.MessageListViewStyle
 import io.getstream.chat.android.client.models.Attachment
 import io.getstream.chat.android.client.models.Message
 
-class AttachmentViewHolder(resId: Int, parent: ViewGroup) :
-    BaseAttachmentViewHolder(resId, parent) {
-
-    private val cl_attachment_media: ConstraintLayout =
-        itemView.findViewById(R.id.cl_attachment_media)
-    private val cl_des: ConstraintLayout =
-        itemView.findViewById(R.id.cl_des)
-    private val iv_media_thumb: PorterShapeImageView =
-        itemView.findViewById(R.id.iv_media_thumb)
-    private val lv_attachment_file: ListView =
-        itemView.findViewById(R.id.lv_attachment_file)
-    private val tv_media_title: TextView =
-        itemView.findViewById(R.id.tv_media_title)
-    private val tv_media_play: TextView =
-        itemView.findViewById(R.id.tv_media_play)
-    private val tv_media_des: TextView =
-        itemView.findViewById(R.id.tv_media_des)
+class AttachmentViewHolder(
+    parent: ViewGroup,
+    private val binding: StreamItemAttachmentBinding =
+        StreamItemAttachmentBinding.inflate(parent.inflater, parent, false)
+) : BaseAttachmentViewHolder(binding.root) {
+
+    private val mediaBinding: StreamItemAttachMediaBinding = binding.clAttachmentMedia
```

Comment:
Stores the binding for the included layout, so that it's easier to access, without having to go through `binding.clAttachmentMedia` for each View.